package.json

```diff
@@ -25,7 +25,8 @@
   },
   "homepage": "https://github.com/FormidableLabs/webpack-dashboard#readme",
   "peerDependencies": {
-    "webpack": "*"
+    "webpack": "*",
+    "webpack-dev-server": "*"
```

Comment:
My bad. If it's used at all then perhaps I should remove it from devDependencies as well?